src/sentry/api/endpoints/project_stacktrace_link.py

```diff
@@ -1,22 +1,24 @@
 import logging
 from typing import Dict, List, Mapping, Optional
 
+import requests
 from rest_framework.request import Request
 from rest_framework.response import Response
 from sentry_sdk import Scope, configure_scope
 
-from sentry import analytics, features
+from sentry import analytics, features, options
 from sentry.api.base import region_silo_endpoint
 from sentry.api.bases.project import ProjectEndpoint
 from sentry.api.serializers import serialize
 from sentry.integrations import IntegrationFeatures
-from sentry.integrations.utils.codecov import Codecov
+from sentry.integrations.utils.codecov import get_codecov_line_coverage
 from sentry.models import Integration, Project, RepositoryProjectPathConfig
 from sentry.shared_integrations.exceptions import ApiError
 from sentry.utils.event_frames import munged_filename_and_frames
 from sentry.utils.json import JSONData
 
 logger = logging.getLogger(__name__)
+CODECOV_TOKEN = options.get("codecov.client-secret")
```

Comment:
You can directly import `CODECOV_TOKEN` from `sentry.integrations.utils.codecov` and you will not need to do this here.